docs/modules/ROOT/pages/deploy-management-center.adoc

```diff
@@ -0,0 +1,210 @@
+
+WARNING: `hazelcastClusters` field does not support deleting clusters from the custom resource. If you want to remove a cluster from the Management Center, you can do it from the Management Center UI.
+
+
+. Apply it with the following command to start Management Center.
++
+.For Kubernetes
+[source,shell]
+----
+kubectl apply -f management-center.yaml
+----
++
+.For Openshift
+[source,shell]
+----
+oc apply -f management-center.yaml
+----
+
+
+. After a moment, you can verify that Management Center is up and running by checking the Management Center logs.
++
+.For Kubernetes
+[source,shell]
+----
+kubectl logs pod/managementcenter-sample-0
+----
++
+.For Openshift
+[source,shell]
+----
+oc logs pod/managementcenter-sample-0
+----
+
+```
+2021-08-26 15:21:04,842 [ INFO] [MC-Client-dev.lifecycle-1] [c.h.w.s.MCClientManager]: MC Client connected to cluster dev.
+2021-08-26 15:21:05,241 [ INFO] [MC-Client-dev.event-1] [c.h.w.s.MCClientManager]: Started communication with member: Member [10.36.8.3]:5701 - ccf31703-de3b-4094-9faf-7b5d0dc145b2
+2021-08-26 15:21:05,245 [ INFO] [MC-Client-dev.event-1] [c.h.w.s.MCClientManager]: Started communication with member: Member [10.36.7.2]:5701 - e75bd6e2-de4b-4360-8113-040773d858b7
+2021-08-26 15:21:05,251 [ INFO] [MC-Client-dev.event-1] [c.h.w.s.MCClientManager]: Started communication with member: Member [10.36.6.2]:5701 - c3d105d2-0bca-4a66-8519-1cacffc05c98
+2021-08-26 15:21:07,234 [ INFO] [main] [c.h.w.Launcher]: Hazelcast Management Center successfully started at http://localhost:8080/
+```
+
+--
+====
+
+To access the Management Center dashboard, open the browser at address `http://$MANCENTER_IP:8080`.
+
+[tabs]
+====
+Kubernetes::
++
+--
+[source,shell]
+----
+MANCENTER_IP=$( kubectl get service managementcenter-sample -o jsonpath='{.status.loadBalancer.ingress[0].ip}')
+----
+--
+Openshift::
++
+--
+[source,shell]
+----
+MANCENTER_IP=$( oc get service managementcenter-sample -o jsonpath='{.status.loadBalancer.ingress[0].ip}')
+----
+--
+====
+
+If EXTERNAL-IP of the service is hostname, not IP, you can run command below:
+
+[tabs]
+====
+Kubernetes::
++
+--
+[source,shell]
+----
+MANCENTER_IP=$( kubectl get service managementcenter-sample -o jsonpath='{.status.loadBalancer.ingress[0].hostname}')
+----
+--
+Openshift::
++
+--
+[source,shell]
+----
+MANCENTER_IP=$( oc get service managementcenter-sample -o jsonpath='{.status.loadBalancer.ingress[0].hostname}')
+----
+--
+====
+
+== Step 4. Clean up
+
+You can run the commands below to remove the Management Center.
+
+[tabs]
+====
+Kubernetes::
++
+--
+[source,shell]
+----
+kubectl delete -f management-center.yaml
+----
+--
+Openshift::
++
+--
+[source,shell]
+----
+oc delete -f management-center.yaml
+----
+--
+====
+
```

Comment:
As we discussed before, some references for `Management Center` section might be good in this page.

Reply:
I realized that we can not provide a link for `Management Center` section directly because it is not a title. You can ignore my comment 👍